openpype/settings/defaults/project_anatomy/attributes.json

```diff
@@ -19,8 +19,7 @@
         "blender/2-91",
         "harmony/20",
         "photoshop/2021",
-        "aftereffects/2021",
-        "unreal/4-26"
```

Comment:
This seems like an odd change for this PR.

Reply:
this is perhaps mainly because I work on an old version, I will pull from the github and push another one again.